source/tests/CMakeLists.txt

```diff
@@ -372,10 +376,12 @@ if(TIMEMORY_USE_PTL OR TARGET PTL::ptl)
         LINK_LIBRARIES  common-test-libs
                         PTL::ptl
                         ${_LIBRARY}
-                        test-debug-flags
                         timemory::timemory-sanitizer
```

Comment:
```suggestion
                        $<$<NOT:$<BOOL:WIN32>>:test-debug-flags>
                        timemory::timemory-sanitizer
```

Reply:
This works fine and is much nicer than the ugly ``if`` logic. 
(Sorry, I am not very good at cmake generator expressions. 
What you did, is quite nice, but not in my normal cmake repretoire.)